tests/foreman/api/test_errata.py

```diff
@@ -513,8 +476,62 @@ def test_positive_filter_by_envs(module_org):
     assert len(errata_library) > len(errata_env)
 
 
+@pytest.fixture(scope='module')
+def setup_content_rhel6():
+    org = entities.Organization().create()
+    with manifests.clone() as manifest:
+        upload_manifest(org.id, manifest.content)
+
+    rh_repo_id_rhva = enable_rhrepo_and_fetchid(
+        basearch='x86_64',
+        org_id=org.id,
+        product=constants.PRDS['rhel'],
+        repo=constants.REPOS['rhva6']['name'],
+        reposet=constants.REPOSET['rhva6'],
+        releasever=constants.DEFAULT_RELEASE_VERSION,
+    )
+    rh_repo = entities.Repository(id=rh_repo_id_rhva).read()
+    rh_repo.sync()
+
+    host_tools_product = entities.Product(organization=org).create()
+    host_tools_repo = entities.Repository(
+        product=host_tools_product,
+    ).create()
+    host_tools_repo.url = settings.repos.SATTOOLS_REPO.RHEL6
+    host_tools_repo = host_tools_repo.update(['url'])
+    host_tools_repo.sync()
+
+    custom_product = entities.Product(organization=org).create()
+    custom_repo = entities.Repository(
+        product=custom_product,
+    ).create()
+    custom_repo.url = CUSTOM_REPO_URL
+    custom_repo = custom_repo.update(['url'])
+    custom_repo.sync()
+
+    lce = entities.LifecycleEnvironment(organization=org).create()
+
+    cv = entities.ContentView(
+        organization=org,
+        repository=[rh_repo_id_rhva, host_tools_repo.id, custom_repo.id],
+    ).create()
+    cv.publish()
+    cvv = cv.read().version[0].read()
+    promote(cvv, lce.id)
+
+    ak = entities.ActivationKey(content_view=cv, organization=org, environment=lce).create()
+
+    sub_list = [DEFAULT_SUBSCRIPTION_NAME, host_tools_product.name, custom_product.name]
+    for sub_name in sub_list:
+        subscription = entities.Subscription(organization=org).search(
+            query={'search': f'name="{sub_name}"'}
+        )[0]
+        ak.add_subscriptions(data={'subscription_id': subscription.id})
+    return ak, org, sub_list
```

Comment:
Returning dictionary instead will look sane ! 

But its not a blocking comment !

Reply:
Followup PR